docs/getting-started/azure-to-cluster/configure.rst

```diff
@@ -0,0 +1,85 @@
+.. _configure-azure-to-cluster:
+
+=================================
+Configure and deploy your cluster
+=================================
+
+After having subscribed to the offer, you can now start the process to deploy
+the cluster.
+
+
+.. rubric:: Table of contents
+
+.. contents::
+   :local:
+
+
+.. _configure-azure-to-cluster_wizard:
+
+Configuration
+=============
+
+First it must be configured. On the *Home > Software as a Service*
+*(SaaS)* page, go to *Overview*. Now click on *Configure Account* at the top
+left of the Overview page.
+
+.. image:: ../../_assets/img/ConfigureAccount.png
```

Comment:
can you switch the image names to use foo-bar.png style naming instead of FooBar.png style please